### ISSUE TITLE
Can't delete just copied file

### ISSUE THREAD

**Issue by ydanila** [coderaiser/cloudcmd#160]
Just updated my cloudcmd from 6.16 to 9.0.0 and faced one funny problem - can't delete just copied file.

I hadn't such problem before update.

Original files
```
drwxr-xr-x  2 ubuntu deluge    4096 Feb 29  2016 .
drwxr-xr-x 14 ubuntu deluge    4096 Jan  7 09:27 ..
-rwxr-xr-x  1 ubuntu deluge 5673924 Jan  7 09:27 ATB - Don't Stop (Pick Up Club Mix).mp3
-rwxr-xr-x  1 ubuntu deluge 6320128 Jan  7 09:27 ATB - Ecstasy.mp3
-rwxr-xr-x  1 ubuntu deluge 3526656 Jan  7 09:27 ATB feat. York - 01 - The Fields Of Love (Radio Edit).mp3
-rwxr-xr-x  1 ubuntu deluge 3465428 Jan  7 09:27 ATB_(The_Summer).mp3
-rwxr-xr-x  1 ubuntu deluge 4908483 Jan  7 09:27 Atb - You're Not Alone.mp3

```

Copy result

```
drwxr-xr-x  2 ubuntu deluge    4096 Feb 29  2016 .
drwxr-xr-x 14 ubuntu deluge    4096 Jan  7 09:27 ..
-rwxr-xr-x  1 ubuntu deluge 5673924 Jan  7 09:27 ATB - Don't Stop (Pick Up Club Mix).mp3
-rwxr-xr-x  1 ubuntu deluge 6320128 Jan  7 09:27 ATB - Ecstasy.mp3
-rwxr-xr-x  1 ubuntu deluge 3526656 Jan  7 09:27 ATB feat. York - 01 - The Fields Of Love (Radio Edit).mp3
-rwxr-xr-x  1 ubuntu deluge 3465428 Jan  7 09:27 ATB_(The_Summer).mp3
-rwxr-xr-x  1 ubuntu deluge 4908483 Jan  7 09:27 Atb - You're Not Alone.mp3

```

* **Version** (`cloudcmd -v`): 
v9.0.0
* **Node Version** `node -v`: 
v6.11.4
* **OS** (`uname -a` on Linux): 
Linux cubox-i 4.9.12-cubox #7 SMP Thu Feb 23 21:22:20 CET 2017 armv7l armv7l armv7l GNU/Linux
* **Browser name/version**: 
Chrome 65.0.3325.181
* **Used Command Line Parameters**: 
* **Changed Config**:

**Comment by coderaiser** [comment]
Looks like it is the same problem with cache is https://github.com/coderaiser/cloudcmd/issues/161. You can disable `cache` in `config dialog`. To avoid such problems in a future.

**Comment by ydanila** [comment]
For this case it opens confirmation dialog for F8 press but nothing happens. Files still exists. 

Where I can find log for cloudcmd?

**Comment by coderaiser** [comment]
Do you have `auth` enabled? Did you change `config?

**Comment by ydanila** [comment]
Here is my config.
```
{
    "name": "",
    "auth": false,
    "username": "root",
    "password": "2b64f2e3f9fee1942af9ff60d40aa5a719db33b8ba8dd4864bb4f11e25ca2bee00907de32a59429602336cac832c8f2eeff5177cc14c864dd116c8bf6ca5d9a9",
    "algo": "sha512WithRSAEncryption",
    "editor": "edward",
    "packer": "tar",
    "diff": true,
    "zip" : true,
    "localStorage": true,
    "buffer": true,
    "dirStorage": false,
    "online": true,
    "open": false,
    "cache": true,
    "keysPanel": true,
    "port": 8000,
    "ip": null,
    "root": "/",
    "prefix": "",
    "progress": true,
    "htmlDialogs": true,
    "contact": true,
    "confirmCopy": true,
    "confirmMove": true,
    "configDialog": true,
    "onePanelMode": false,
    "console": true,
    "syncConsolePath": false,
    "terminal": false,
    "terminalPath": "",
    "showConfig": false,
    "vim": false,
    "columns": "name-size-date-owner-mode"
}

```

**Comment by coderaiser** [comment]
OK, I got it. This can be related to `auth` flag.

**Comment by coderaiser** [comment]
Thank you, fixed with f248e29, landed in [v9.1.0](https://github.com/coderaiser/cloudcmd/releases/tag/v9.1.0).
